sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -319,3 +325,31 @@ def test_repr_kernels():
 
     for kernel in kernels:
         repr(kernel)
+
+
+def test_select_dimension_kernel_properties():
+    kernel1 = RBF(length_scale=[1.0, 2.0])
+    kernel2 = Matern(length_scale=[2.0, 3.0])
+    sdk1 = SelectDimensionKernel(kernel=kernel1, active_dims=[0, 1])
```

Comment:
I can't see a test where the indices are specified as a numpy array already.

I also can't see a test where the indices are specified out of order.

I also can't see a test where the mask is all false or indices are empty.